src/Command/ColdStorageFlagCommand.php

```diff
@@ -78,78 +73,45 @@ protected function configure()
-
-            if (file_exists($stubFileName)) {
-                $output->writeln("UDI: ($udi)");
-                $output->writeln("Original Size: ($size)");
-                $output->writeln("Original Hash: ($hash)");
-                $output->writeln("Original Filename: ($originalFilename)");
-                $output->writeln("stubFileName: ($stubFileName)");
-                $output->writeln("Attempting to flag $udi as Cold Stored.");
-
-                $datasets = $this->entityManager->getRepository(Dataset::class)->findBy(array('udi' => $udi));
-
-                if (count($datasets) == 0) {
-                    throw new \Exception('Could not find a dataset with the udi provided.');
-                } else {
-                    $output->writeln('Dataset Found.');
-                }
-                $dataset = $datasets[0];
-
-                $datasetSubmission = $dataset->getDatasetSubmission();
-                if (!($datasetSubmission instanceof DatasetSubmission)) {
-                    throw new \Exception('Could not find Dataset Submission.');
-                } else {
-                    $output->writeln('Submission Found.');
-
-                    // Set Modifier
-                    $datasetSubmission->setModifier($systemPerson);
-
-                    // Set filesize, hash, and original filename. (Cold Storage attributes).
-                    $datasetSubmission->setDatasetFileColdStorageAttributes($size, $hash, $originalFilename);
-
-                    // Set options for a new replacement datafile of the supplied Cold-Storage stubfile.
-                    $datasetSubmission->setDatasetFileTransferStatus(DatasetSubmission::TRANSFER_STATUS_NONE);
-                    $datasetSubmission->setDatasetFileName(null);
-                    $datasetSubmission->setDatasetFileSha256Hash(null);
-                    $datasetSubmission->setDatasetFileTransferType(DatasetSubmission::TRANSFER_TYPE_SFTP);
-                    $datasetSubmission->setDatasetFileUri($stubFileName);
-
-                    $this->entityManager->persist($datasetSubmission);
-                    $this->entityManager->flush();
-
-                    //Use rabbitmq to process dataset file and persist the file details. This will
-                    //Trigger filer and hasher (via filer) to complete the process.
-                    $datasetSubmissionFilerMessage = new DatasetSubmissionFiler($datasetSubmission->getId());
-                    $this->messageBus->dispatch($datasetSubmissionFilerMessage);
-                }
-            } else {
-                throw new \Exception("Could not open $stubFileName, expected to be at same location as $infoFileName.");
-            }
+        $io = new SymfonyStyle($input, $output);
+        $udi = $input->getOption('udi');
+        $size = $input->getOption('originalfilesize');
+        $hash = $input->getOption('originalfilehash');
+        $originalFilename = $input->getOption('originalfilename');
```

Comment:
I would keep the variables somewhat consistent, just like `originalFilename` (besides caps) it's the same everywhere.